test/unit/extract/xbrl_test.py

```diff
@@ -135,68 +134,6 @@ def test_xbrl2sqlite(settings, forms, mocker, tmp_path):
         )
 
 
-def test_xbrl2sqlite_db_exists_no_clobber(mocker, tmp_path):
-    convert_form_mock = mocker.MagicMock()
-    mocker.patch("pudl.extract.xbrl.convert_form", new=convert_form_mock)
-
-    # Mock datastore object to allow comparison
-    mock_datastore = mocker.MagicMock()
-    mocker.patch("pudl.extract.xbrl.FercXbrlDatastore", return_value=mock_datastore)
-
-    ferc1_sqlite_path = PudlPaths().output_dir / "ferc1_xbrl.sqlite"
-    ferc1_sqlite_path.touch()
-    settings = FercToSqliteSettings(
-        ferc1_xbrl_to_sqlite_settings=Ferc1XbrlToSqliteSettings(),
-    )
-    context = build_op_context(
-        resources={
-            "ferc_to_sqlite_settings": settings,
-            "datastore": "datastore",
-            "runtime_settings": RuntimeSettings(
-                clobber=False,
-                xbrl_batch_size=20,
-                xbrl_num_workers=10,
-            ),
-        },
-    )
-
-    assert ferc1_sqlite_path.exists()
-    with pytest.raises(RuntimeError, match="Found existing DB"):
-        xbrl2sqlite_op_factory(XbrlFormNumber.FORM1)(context)
-    assert ferc1_sqlite_path.exists()
-
-
-def test_xbrl2sqlite_db_exists_yes_clobber(mocker, tmp_path):
-    convert_form_mock = mocker.MagicMock()
-    mocker.patch("pudl.extract.xbrl.convert_form", new=convert_form_mock)
-
-    # Mock datastore object to allow comparison
-    mock_datastore = mocker.MagicMock()
-    mocker.patch("pudl.extract.xbrl.FercXbrlDatastore", return_value=mock_datastore)
-
-    ferc1_sqlite_path = PudlPaths().output_dir / "ferc1_xbrl.sqlite"
-    ferc1_sqlite_path.touch()
-    settings = FercToSqliteSettings(
-        ferc1_xbrl_to_sqlite_settings=Ferc1XbrlToSqliteSettings(),
-    )
-
-    context = build_op_context(
-        resources={
-            "ferc_to_sqlite_settings": settings,
-            "datastore": "datastore",
-            "runtime_settings": RuntimeSettings(
-                clobber=True,
-                xbrl_batch_size=20,
-                xbrl_num_workers=10,
-            ),
-        },
-    )
-
-    assert ferc1_sqlite_path.exists()
-    xbrl2sqlite_op_factory(XbrlFormNumber.FORM1)(context)
-    assert not ferc1_sqlite_path.exists()
-
-
```

Comment:
My interpretation of these is tests is that they were only checking the clobber/no-clobber behavior, which should no longer be necessary.

Reply:
Yeah, and they also occasionally screwed up nightly builds for complicated reasons. 🔥